app/index.js

```diff
@@ -33,16 +24,17 @@ function handleOpenURL(url) {
   }
 }
 
+
 Linking.addEventListener('url', event => handleOpenURL(event.url));
 
 
 export default class App extends Component {
 
   componentDidMount() {
```

Comment:
i think we should move this to a saga! i think it s bad practice to use the view to trigger it. What do you think?